client/src/components/SolutionViewer/SolutionViewer.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { Panel } from '@freecodecamp/react-bootstrap';
+import Prism from 'prismjs';
+import React from 'react';
+
+interface solutionviewerProps {
```

Comment:
```suggestion
type SolutionViewerProps = {
```
PascalCase, also don't believe this needs to be an interface.